test/unit/GridView.spec.js

```diff
@@ -0,0 +1,112 @@
+import { mount } from '@vue/test-utils'
+import Vue from 'vue'
+import GridView from '@/components/GridView.vue'
+import { CardTypes } from '@/types/Card'
+
+const generateSkeletons = (n) =>
+  Array(n).fill({
+    type: CardTypes.SKELETON
+  })
+
+const mountWithProps = (component, props = {}) => {
+  return mount(component, {
+    propsData: {
+      value: 152,
+      columns: 3,
+      loading: false,
+      didShowMore: false,
+      isMobile: false,
+      width: 800,
+      cards: generateSkeletons(10),
+      unusedCards: [],
+      emptySlots: 5,
+      loadFull: jest.fn(),
+      ...props
+    }
+  })
+}
+
+describe('Grid.spec.js', () => {
+  test('snapshots', () => {
+    // On desktop
+    const wrapper = mountWithProps(GridView)
+    expect(wrapper.element).toMatchSnapshot()
+
+    // On mobile
+    const wrapper2 = mountWithProps(GridView, {
+      isMobile: true
+    })
+    expect(wrapper2.element).toMatchSnapshot()
+
+    // On mobile with more content shown
+    const wrapper3 = mountWithProps(GridView, {
+      isMobile: true,
+      didShowMore: true
+    })
+    expect(wrapper3.element).toMatchSnapshot()
+  })
```

Comment:
@benjick Do you see any specific need for snapshots? Because they will break on any code change, right? And they need to be regenerated on commit?